src/frontend/src/lib/types/token.ts

```diff
@@ -66,3 +66,13 @@ interface TokenFinancialData {
 }
 
 export type TokenUi = Token & TokenFinancialData;
+
+export type TokenUiWithLinkedData = TokenUi & TokenFinancialData & RequiredTokenWithLinkedData;
+
+export interface TokenGroupUi {
+	header: TokenMetadata;
+	native: Network;
```

Comment:
`nativeNetwork`?

Reply:
- `metadata`?
- we need the all nerwork information - not just the ID?

Reply:
@darioloetsc any news on the above?

Reply:
459473a4

yes, the network information is needed in the view component later on